sematic/db/db.py

```diff
@@ -20,7 +20,19 @@ class DB:
     """
 
     def __init__(self, url: str):
-        self._engine = sqlalchemy.create_engine(url)
+        self._engine = sqlalchemy.create_engine(
```

Comment:
In order to validate that this actually solves #792, please perform this test:
- fwd postgres port locally
- configure local sematic instance to use the local postgres port
- start pipeline
- kill port fwding
- wait until you start getting connection errors
- fwd postgres port locally
- confirm that pipeline execution finishes successfully

Reply:
Executed the suggested test. The pipeline did not continue after breaking the connections, but the server was able to continue serving other requests once the port forwarding was restored. This is enough evidence for me that the server is able to pick up & restore connections following a temporary DB outage.